bukkit/rpk-permissions-bukkit/src/main/kotlin/com/rpkit/permissions/bukkit/command/charactergroup/CharacterGroupListCommand.kt

```diff
@@ -0,0 +1,62 @@
+package com.rpkit.permissions.bukkit.command.charactergroup
+
+import com.rpkit.characters.bukkit.character.RPKCharacterService
+import com.rpkit.core.service.Services
+import com.rpkit.permissions.bukkit.RPKPermissionsBukkit
+import com.rpkit.permissions.bukkit.group.RPKGroupService
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+
+class CharacterGroupListCommand(private val plugin: RPKPermissionsBukkit) : CommandExecutor {
+
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
+        if (sender !is Player) {
+            sender.sendMessage(plugin.messages["not-from-console"])
+            return true
+        }
+        if (!sender.hasPermission("rpkit.permissions.command.character.group.list")) {
+            sender.sendMessage(plugin.messages["no-permission-character-group-list"])
+            return true
+        }
+
+        if (args.isEmpty()) {
+            sender.sendMessage(plugin.messages["no-character"])
+            return true
+        }
+
+        val rpkGroupService = Services[RPKGroupService::class.java]
+        if (rpkGroupService == null) {
+            sender.sendMessage(plugin.messages["no-character-service"])
```

Comment:
The error message here should reference the group service
```suggestion
            sender.sendMessage(plugin.messages["no-group-service"])
```